flocker/control/script.py

```diff
@@ -77,6 +81,35 @@ def main(self, reactor, options):
 
 
 def flocker_control_main():
+    pr = cProfile.Profile()
+
+    def enable_profiling(signal, frame):
+        """
+        Enable profiling of the control service.
+
+        :param int signal: See ``signal.signal``.
+        :param frame: None or frame object. See ``signal.signal``.
+        """
+        pr.enable()
+
+    def output_profile(signal, frame):
```

Comment:
:question: As for the docs, consider renaming to `disable_profiling` to emphasise on/off semantics, and document profile output as a side effect.